src/App/analytics.ts

```diff
@@ -0,0 +1,61 @@
+import GoogleAnalytics, { resetClientId } from 'electron-ga-uuid';
+import { reaction } from 'mobx';
+import { AnalyticsInterface } from '../../public/models/analytic.model';
+import isDev from 'electron-is-dev';
+
+const DEV_TRACK_ID = 'UA-169320344-1';
+const TRACK_ID = 'UA-22170471-17';
+
+export default class Analytics implements AnalyticsInterface {
+  constructor(settings: { enableAnalytics: boolean }) {
+    this.isEnabled = !isDev && settings.enableAnalytics;
+    this.ga = null;
```

Comment:
It's best not to use `null` in TS - use `undefined` where possible. You can remove this assignment if you make the `ga` property optional.

Reply:
removed.